task_spec/spec/acceptance/init_spec.rb

```diff
@@ -50,8 +51,20 @@ def add_node(node, group_name, groups)
           config: config
         }
       else
-        # This is very unlikely to work
-        node_name, node = host.hostname
+        # host_hash is differnet depending on hypervisor (docker vs vmpooler)
```

Comment:
Can we decide how to process the host_hash with a conditional on the hypervisor directly